src/components/cohorts/CohortPage.js

```diff
@@ -1,27 +1,82 @@
 import Header from '../Header/Header';
 import client from '../../utils/client';
-import { useParams } from 'react-router-dom';
+import { useParams, useNavigate } from 'react-router-dom';
 import { useState, useEffect } from 'react';
-
+import './style.css';
 
 const CohortPage = () => {
   const params = useParams();
-  const [cohort, setCohort] = useState()
+  const navigate = useNavigate()
+  const [cohortName, setCohortName] = useState()
+  const [studentsInCohort, setStudentsInCohort] = useState()
+  const [availableUsers, setAvailableUsers] = useState()
 
   useEffect(() => {
     client
-        .get(`/cohort/${params.id}`)
+        .get(`/cohort/${params.id}?availableStudents=true`)
         .then((res) => {
-          const cohort = res.data.data
-          setCohort(cohort)
+          setCohortName(res.data.data.cohortName)
+          setStudentsInCohort(res.data.data.users)
+          setAvailableUsers(res.data.data.availableStudents)
         })
         .catch((err) => console.error(err.response));
   }, [params]);
 
+  const handleSubmitAddStudentToCohort = (e, userId) => {
+    e.preventDefault();
+    client
+        .patch(`/user/${userId}`, { cohort_id: +params.id })
```

Comment:
`+params.id` - is this converting params.id to an integer? Would there be any conditions where this fails? In which case you might want to do the conversion to int and test it was converted successfully before submitting the fetch request.

Reply:
yes it gets updated to a number, since weve got error handling client and server side this shouldnt be an issue